### PR TITLE
chore: [IOBP-535] Added german language for iOS app

### PR DESCRIPTION
**Pull request by mastro993**
## Short description
This PR adds the german language in the supported languages list in the App Store page.

## List of changes proposed in this pull request
- Added `de` locale to iOS project files

## How to test
n/a

### CODE REVIEW

File: ios/ItaliaApp/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"ALERT_DEVICE_ROOTED_TITLE" = "Device jailbroken";
```
Comment (thisisjp) [inline ios/ItaliaApp/de.lproj/Localizable.strings:1]: @dhinterlechner could you please help us with the translation of these 2 strings? 🙏

Comment (dhinterlechner) [inline ios/ItaliaApp/de.lproj/Localizable.strings:1]: Gerät mit Jailbreak

File: ios/ItaliaApp/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+"ALERT_DEVICE_ROOTED_DESC" = "This device is jailbroken, you can't use this app for security reasons";
```
Comment (dhinterlechner) [inline ios/ItaliaApp/de.lproj/Localizable.strings:2]: Dieses Gerät ist gejailbreakt, du kannst diese App aus Sicherheitsgründen nicht verwenden

File: ios/ItaliaApp/de.lproj/Localizable.strings (hunk 1 above)
Comment (thisisjp) [inline ios/ItaliaApp/de.lproj/Localizable.strings:1]:
```suggestion
"ALERT_DEVICE_ROOTED_TITLE" = "Gerät mit Jailbreak";
```

File: ios/ItaliaApp/de.lproj/Localizable.strings (hunk 2 above)
Comment (thisisjp) [inline ios/ItaliaApp/de.lproj/Localizable.strings:2]:
```suggestion
"ALERT_DEVICE_ROOTED_DESC" = "Dieses Gerät ist gejailbreakt, du kannst diese App aus Sicherheitsgründen nicht verwenden";
```

**Review by thisisjp — CHANGES_REQUESTED** [review]

**Review by thisisjp — APPROVED** [review]

**Comment by mastro993** [comment]
@dhinterlechner  we also need a translation for these strings. Do you mind translating them, please?

```strings
"NSCameraUsageDescription" = "Potrai leggere codici QR e pagare gli avvisi pagoPA.";
"NSCalendarsUsageDescription" = "Se acconsenti, potrai aggiungere le scadenze al tuo calendario e impostare un promemoria.";
"NSContactsUsageDescription" = "IO ha necessità di accedere ai tuoi contatti per permetterti di aggiungerli come partecipanti alle scadenze";
"NSFaceIDUsageDescription" = "IO ha necessità di accedere a Face ID per permetterti di effettuare un'autenticazione più veloce";
"NSPhotoLibraryUsageDescription" = "Potrai salvare bonus e certificati, caricare screenshot e avvisi di pagamento.";
"NSMicrophoneUsageDescription" = "IO ha necessità di accedere al microfono nel caso in cui desideri mandare una nota vocale";
"NSPhotoLibraryAddUsageDescription" = "Potrai salvare immagini dall'app sul tuo dispositivo.";
```

cc: @thisisjp

**Review by dhinterlechner — APPROVED** [review]
👍

**Review by shadowsheep1 — APPROVED** [review]
LGTM

**Pull request merged**